platformio.ini

```diff
@@ -0,0 +1,70 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[platformio]
+src_dir = main
+
+[env]
+build_flags = 
+    -Iinclude/tls_config
+    -DFRAMEWORK_$PIOFRAMEWORK
+
+[env:esp32dev]
+platform = espressif32
+framework = espidf
+board = esp32dev
+build_flags = 
+    ${env.build_flags}
+    -Iinclude
+    -DMBEDTLS_USER_CONFIG_FILE=\"mbedtls+HKDF.h\"
+board_build.embed_files =
+    components/ena/test/export.bin
+    components/ena-cwa/certs/telekom.pem
```

Comment:
CMakeLists.txt are parsed differently than in esp-idf. This is compensated here.

-Iinclude  -DMBEDTLS_USER_CONFIG_FILE=\"mbedtls+HKDF.h\"
is used to enable HKDF.

board_build.embed_files is PlatformIOs way for including verbatim file contents into build result.